com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -13,7 +13,7 @@ class MessageBatcher
     {
         public class SendStream
         {
-            public string channel;
+            public byte channel;
```

Comment:
"256 channels ought to be enough for everyone" ? [640K](https://www.computerworld.com/article/2534312/the--640k--quote-won-t-go-away----but-did-gates-really-say-it-.html)

I'm not arguing for more, I think 256 _is_ enough. But just want to point out the limit and ask whether we stopped to consider the impact

Reply:
Yeah, I wondered about the same thing, and even thought about putting dup channel detection in the registration function just in case.  I thought from our discussion we wanted to stick with 'byte'.  Happy to go all the way to a QWORD too :)

Reply:
I think byte should be fine for the next few milestones, but it could be that when we get into the MMO side of things this value could have potential to be increased to a ushort or even a uint.